Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Temp, not sure what to do here yet");
```

Comment:
Demo should call the functions that you made- specifically test the two people bmi calculation that you did in homework 1. and 2